pages/version-policy/index.md

```diff
@@ -331,31 +331,25 @@ B - This package combination is *beta*.
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Edge-LB 1.2.1</td>
+        <td>Edge-LB 1.2.3</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Edge-LB 1.2.2</td>
+        <td>Edge-LB 1.3.0 (Recommended)</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Edge-LB 1.2.3 (Recommended)</td>
```

Comment:
Looks like on Edge-LB version is taken off. It should be Edge-LB v1.3.0(recommended), Edge-LB v.1.2.3, and Edge-LB v1.2.2